pkg/manager/member/utils.go

```diff
@@ -173,6 +173,29 @@ func CombineAnnotations(a, b map[string]string) map[string]string {
 	return a
 }
 
+func CopyAnnotations(src map[string]string) map[string]string {
+	if src == nil {
+		return nil
+	}
+	dst := map[string]string{}
+	for k, v := range src {
+		dst[k] = v
+	}
+	return dst
+}
+
+// MergeLabels merge additional labelset and base labelset to a new labelset. Additional labelset will override base labelset if conflicts.
+func MergeLabels(base, additional map[string]string) map[string]string {
```

Comment:
This util function seems not dedicated to labels, but general `map[string]string` types.
Maybe rename it to a more general name?